modules/draco/src/lib/draco-module-loader.ts

```diff
@@ -9,11 +9,15 @@ const DRACO_ENCODER_VERSION = '1.4.1';
 
 const STATIC_DECODER_URL = `https://www.gstatic.com/draco/versioned/decoders/${DRACO_DECODER_VERSION}`;
 
-const DRACO_JS_DECODER_URL = `${STATIC_DECODER_URL}/draco_decoder.js`;
-const DRACO_WASM_WRAPPER_URL = `${STATIC_DECODER_URL}/draco_wasm_wrapper.js`;
-const DRACO_WASM_DECODER_URL = `${STATIC_DECODER_URL}/draco_decoder.wasm`;
+export const DRACO_JS_DECODER_NAME = 'draco_decoder.js';
+export const DRACO_JS_DECODER_URL = `${STATIC_DECODER_URL}/${DRACO_JS_DECODER_NAME}`;
+export const DRACO_WASM_WRAPPER_NAME = 'draco_wasm_wrapper.js';
+export const DRACO_WASM_WRAPPER_URL = `${STATIC_DECODER_URL}/${DRACO_WASM_WRAPPER_NAME}`;
+export const DRACO_WASM_DECODER_NAME = 'draco_decoder.wasm';
+export const DRACO_WASM_DECODER_URL = `${STATIC_DECODER_URL}/${DRACO_WASM_DECODER_NAME}`;
```

Comment:
Nit: 
- Mostly from an aesthetic point of view, I feel this is a lot of top-level exports for a rather rare use case. How about we group them in name space objects and we export those.
- Also while these names preserve the underlying names, they are not easy to understand for end users. I think we can choose better names. Ideally it should be immediately clear to the user who reads this code what the primary and fallback decoder JS libs are:

```suggestion
export const DRACO_EXTERNAL_LIBRARIES = {
  /** The primary Draco3D encoder, javascript wrapper part */
  DECODER = 'draco_wasm_wrapper.js',
  /** The primary draco encoder, compiled web assembly part */
  DECODER_WASM = draco_wasm_wrapper.js',
  /** Fallback decoder for non-webassebly environments. Very big bundle, lower performance */
  FALLBACK_DECODER: 'draco_decoder.js',  
  /** Draco encoder */
  ENCODER: 'draco_encoder.js',
}

export const DRACO_EXTERNAL_LIBRARY_URLS = {
  [DRACO_EXTERNAL_LIBRARIES. DECODER]: `${STATIC_DECODER_URL}/${DRACO_WASM_WRAPPER_NAME}`;
  [DRACO_EXTERNAL_LIBRARIES]. DECODER_WASM]: `${STATIC_DECODER_URL}/${DRACO_WASM_DECODER_NAME}`;
  [DRACO_EXTERNAL_LIBRARIES.FALLBACK_DECODER]: `${STATIC_DECODER_URL}/${DRACO_JS_DECODER_NAME}`;
  [...ENCODER}: ...
}
```

Reply:
Done